src/Core/src/Platform/Android/LabelExtensions.cs

```diff
@@ -22,5 +23,63 @@ public static void UpdateTextColor(this TextView textView, ILabel label,  Maui.C
 				textView.SetTextColor(textColor.ToNative());
 			}				
 		}
-	}
+
+        public static void UpdateLineBreakMode(this TextView textView, ILabel label)
+        {
+            textView.SetLineBreakMode(label);
```

Comment:
What happens if I set max lines, then change this. For example, I set MaxLines to 3 and Break to Character. Then when I swap to Word, it is going to reset the max lines to int.MaxValue.